### ISSUE TITLE
0.0.20181218 (FW v2.0) Installation warning on ER-X

### ISSUE THREAD

**Issue by broizter** [Lochnair/vyatta-wireguard#90]
When installing 0.0.20181218 (FW v2.0) to an Edgerouter X running version 2.0.0 you get this warning during install: 

`depmod: WARNING: could not open /lib/modules/4.14.54-UBNT/modules.order: No such file or directory`

WireGuard seems to run without problems regardless though. I did not get the same warning when installing to an Edgerouter 4.

**Comment by TBK** [comment]
Sounds strange.

Can I get you to list the folder contents with `sudo ls -la /lib/modules/4.14.54-UBNT/` ?

**Comment by Lochnair** [comment]
ER-X
```
$ ls -1 squashfs-root/lib/modules/4.14.54-UBNT/
extra
kernel
modules.alias
modules.alias.bin
modules.builtin
modules.builtin.bin
modules.dep
modules.dep.bin
modules.devname
modules.softdep
modules.symbols
modules.symbols.bin
```

vs ERL

```
$ ls -1 squashfs-root/lib/modules/4.9.79-UBNT/
extra
kernel
modules.alias
modules.alias.bin
modules.builtin
modules.builtin.bin
modules.dep
modules.dep.bin
modules.devname
modules.order
modules.softdep
modules.symbols
modules.symbols.bin
```

**Comment by TBK** [comment]
modules.order is also missing from the beta builds, but is part of the 1.10.8 FW so it is related to how UBNT builds the 4.14.54 kernel.

I have reported it on the 2.0.0 FW release thread.

**Comment by aaronstuder** [comment]
I am having the same issue

**Comment by TBK** [comment]
Fixed with 2.0.1:
```
Kernel - Restore "modules.order" file on ER-X which is needed when installing 3rd party kernel modules
```
Source : https://dl.ubnt.com/firmwares/edgemax/v2.0.x/changelog.v2.0.1.txt
